File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int count {1};
        int prev;
        int place{0};
        for(int i=0;i<nums.size();i++){
            if(i==0){
                prev = i;
                continue;
            }
            if(nums[i] == nums[prev]) continue;
            place++;
            nums[place] = nums[i];
            prev = i;
            count++;
        }
        for(int i:nums){
            cout<<i<<endl;
        }
        return count;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        curr=0
        start=1
        for i in range(len(nums)):
            if(nums[i]==nums[curr]):
                continue
            if(nums[i] != nums[curr]):
                start+=1
                nums[curr+1] = nums[i]
                curr+=1
        return start
